cmake/sysbuild/modules/ncs_sysbuild_extensions.cmake

```diff
@@ -47,3 +47,42 @@ function(ExternalNcsVariantProject_Add)
   # Configure variant image after application so that the configuration is present
   sysbuild_add_dependencies(CONFIGURE ${VBUILD_VARIANT} ${VBUILD_APPLICATION})
 endfunction()
+
+# Usage:
+#   ExternalNcsCopyProject_Add(APPLICATION <name>
+#                              COPY <name>
+#   )
+#
+# This function includes a copy of an existing Zephyr based build under
+# a specified name into the multiimage build system
+#
+# APPLICATION <name>: Name of the application which is used as the source for
+#                     the copy build.
+# COPY <name>:        Name of the copy build.
+function(ExternalNcsCopyProject_Add)
+  cmake_parse_arguments(VBUILD "" "APPLICATION;COPY" "" ${ARGN})
+
+  ExternalProject_Get_Property(${VBUILD_APPLICATION} SOURCE_DIR BINARY_DIR)
+  set(${VBUILD_APPLICATION}_BINARY_DIR ${BINARY_DIR})
+
+  get_property(COPY_BOARD TARGET ${VBUILD_APPLICATION} PROPERTY BOARD)
+  if(DEFINED COPY_BOARD)
+    ExternalZephyrProject_Add(
+      APPLICATION ${VBUILD_COPY}
+      SOURCE_DIR ${SOURCE_DIR}
+      BOARD ${COPY_BOARD}
+      BUILD_ONLY true
+  )
+  else()
+    ExternalZephyrProject_Add(
+      APPLICATION ${VBUILD_COPY}
+      SOURCE_DIR ${SOURCE_DIR}
+      BUILD_ONLY true
+    )
+  endif()
+
+  set_property(TARGET ${VBUILD_COPY} PROPERTY NCS_VARIANT_APPLICATION ${VBUILD_APPLICATION})
```

Comment:
```
  set_property(TARGET ${VBUILD_VARIANT} APPEND PROPERTY _EP_CMAKE_ARGS
    -DCONFIG_NCS_IS_VARIANT_IMAGE=y
  )
```
should likely be for this

Reply:
Changed to the flag. The `PRELOAD_BINARY_DIR` variable seems to be used only by the Kconfig override logic, thus I put it under the same condition.

Reply:
Changed to keep the `CONFIG_NCS_IS_VARIANT_IMAGE` symbol for both variant application types.